src/Misc/layoutbin/de-DE/strings.json

```diff
@@ -423,7 +423,6 @@
     "RMCachingAllItems":  "Alle Elemente im Dateicontainer werden zwischengespeichert...",
     "RMCachingComplete":  "Die Zwischenspeicherung wurde abgeschlossen. ({0} ms)",
     "RMCachingContainerItems":  "Elemente werden unter \"{0}\" im Dateicontainer zwischengespeichert...",
-    "RMCommandNotFound":  "##vso[task.{0}] wird nicht als Befehl für die Erweiterung des Befehls \"Release\" erkannt. Schlagen Sie in der Dokumentation (http://go.microsoft.com/fwlink/?LinkId=817296) nach.",
```

Comment:
is this folding into something else?

Reply:
yeah, I added "CommandNotFound2" which will take care of this line in a generic way, but there is a test that will fail if you have a string defined in the strings.json that is not referenced. This one is awesome because the translation was not correct, "task.{0}" instead of "release.{0}".